app/v1/controllers/actions/AuthAction.scala

```diff
@@ -171,7 +164,5 @@ class AuthAction @Inject() (
           loginTimes
         )
         credentialMap(request, block, credentials, data, affinityGroup)
-
-      case _ => authorisedRetrievalFailure
```

Comment:
this case never reach here asked auth team, if something goes wrong it throws exception